### PR TITLE
Fix/sonarqube issues 2

### PR DESCRIPTION
**Pull request by Cees2505**
Fixed some Sonarqube issues

### CODE REVIEW

File: core/src/main/java/com/ritense/valtimo/repository/CamundaSearchProcessInstanceRepository.java [hunk 1]
```diff
@@ -30,6 +30,8 @@
 public class CamundaSearchProcessInstanceRepository {
 
     private final SqlSession session;
+    private final static String SEARCH_INSTANCE_COUNT_STATEMENT = "com.ritense.valtimo.camunda.processinstance.searchInstancesCount";
+    private final static String SEARCH_INSTANCES_STATEMENT = "com.ritense.valtimo.camunda.processinstance.searchInstancesCount";
```
Comment (ThomasMinkeRitense) [inline core/src/main/java/com/ritense/valtimo/repository/CamundaSearchProcessInstanceRepository.java:34]: I think this value is not correct

Comment (theo-ritense) [inline core/src/main/java/com/ritense/valtimo/repository/CamundaSearchProcessInstanceRepository.java:34]:
```suggestion
    private final static String SEARCH_INSTANCES_STATEMENT = "com.ritense.valtimo.camunda.processinstance.searchInstances";
```

File: document-generation/smartdocuments/src/main/kotlin/com/ritense/smartdocuments/io/BaseInputStream.kt
Comment (theo-ritense) [inline document-generation/smartdocuments/src/main/kotlin/com/ritense/smartdocuments/io/BaseInputStream.kt:1]: Please implement a new method and call that from the other methods:
```
private fun checkClosed() {
     if (closed) {
            throw IOException("Stream is closed")
     }
}
```

Comment (Cees2505) [reply document-generation/smartdocuments/src/main/kotlin/com/ritense/smartdocuments/io/BaseInputStream.kt:1]: Created a generic method throwing an IOException now

File: core/src/main/java/com/ritense/valtimo/repository/CamundaSearchProcessInstanceRepository.java (hunk 1 above)
Comment (Cees2505) [reply core/src/main/java/com/ritense/valtimo/repository/CamundaSearchProcessInstanceRepository.java]: Fixed

File: core/src/main/java/com/ritense/valtimo/service/ChoiceFieldValueService.java [hunk 2]
```diff
@@ -113,7 +114,7 @@ public void delete(Long id) {
      */
     @Transactional(readOnly = true)
     public Page<ChoiceFieldValue> findAllByChoiceFieldKeyName(Pageable pageable, String choiceFieldName) {
-        logger.debug("Request to get all ChoiceFieldValues for choiceField {}", choiceFieldName);
+        logger.debug(REQUEST_CHOICEFIELD_VALUES, choiceFieldName);
```
Comment (ivo-ritense) [inline core/src/main/java/com/ritense/valtimo/service/ChoiceFieldValueService.java:117]: Honestly i would change the logged text to reflect what is different. That way you can always find the specfific line in the code, and the information is more accurate
```suggestion
        logger.debug("Request to get all ChoiceFieldValues for choiceField with name '{}'", choiceFieldName);
```

**Review by ivo-ritense — APPROVED** [review]

File: document-generation/smartdocuments/src/main/kotlin/com/ritense/smartdocuments/io/BaseInputStream.kt
Comment (theo-ritense) [reply document-generation/smartdocuments/src/main/kotlin/com/ritense/smartdocuments/io/BaseInputStream.kt:1]: This is worse.

**Pull request merged**